README.md

```diff
@@ -18,6 +18,26 @@ npm start
 
 Which will start the web application
 
+## Static build
+
+If you want a static build of the application you can execute
+
+```bash
+npm run build
+```
+
+This will create a static build in the `build` folder.
+To serve this build you can use [serve](https://www.npmjs.com/package/serve) by executing
+
+```bash
+npm install -g serve
+serve -s build
+```
+
+Which will either host the website on port 3000 or pick a random port if 3000 is already in use.
+If you want to specify a port you can use the `-l` flag proceeded by the port number of your choice. 
+
+
```

Comment:
Remove this because this has nothing to do with how to build the app.

Reply:
Removed it in [commit 402db13](https://github.com/SolidLabResearch/generic-data-viewer/pull/62/commits/402db139419551cb472940c3ea22d759cad0df79)